<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QChrono</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/img/img/clock.png</normaloff>:/img/img/clock.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <widget class="QPushButton" name="closeWindowButton">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{ 
	background-color: rgb(255, 96, 89);
	border-radius:7;
	border-width: 1px;
	border-style: solid;
	border-color: rgb(225, 71, 64);
}
QPushButton:hover{ 
	background-color: rgb(240, 0, 0);
	border-radius:7;
	border-width: 1px;
	border-style: solid;
	border-color: rgb(225, 71, 64);
}
QPushButton:pressed{ 
	background-color: rgb(220, 0, 0);
	border-radius:7;
	border-width: 1px;
	border-style: solid;
	border-color: rgb(225, 71, 64);
}

</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="restoreWindowButton_2">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{ 
	background-color:rgb(255, 189, 46);
	border-radius:7;
	border-width: 1px;
	border-style: solid;
	border-color: rgb(223, 161, 35);
}
QPushButton:hover{ 
	background-color: rgb(255, 255, 0);
	border-radius:7;
	border-width: 1px;
	border-style: solid;
	border-color: rgb(223, 161, 35);
}
QPushButton:pressed{ 
	background-color: rgb(100, 100, 0);
	border-radius:7;
	border-width: 1px;
	border-style: solid;
	border-color: rgb(223, 161, 35);
}

</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="restoreWindowButton">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{ 
	background-color: rgb(41, 206, 66);
	border-radius:7;
	border-width: 1px;
	border-style: solid;
	border-color: rgb(37, 204, 62);
}
QPushButton:hover{ 
	background-color: rgb(0, 255, 0);
	border-radius:7;
	border-width: 1px;
	border-style: solid;
	border-color: rgb(37, 204, 62);
}
QPushButton:pressed{ 
	background-color: rgb(40, 200, 50);
	border-radius:7;
	border-width: 1px;
	border-style: solid;
	border-color: rgb(37, 204, 62);
}

</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="windowTittleLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>QChrono</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>1</number>
        </property>
        <property name="indent">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>       min     :      sec      :     1/100    </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="gobLCD">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>48</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>00:00:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPlainTextEdit" name="gobLapTextArea">
        <property name="font">
         <font>
          <family>Rockwell Condensed</family>
          <pointsize>36</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(43, 43, 43);
color: rgb(85, 255, 0);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="undoRedoEnabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="centerOnScroll">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="clearLapsButton">
          <property name="text">
           <string>&amp;Clear laps</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="startButton">
            <property name="text">
             <string>&amp;Start</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/img/img/Go.png</normaloff>:/img/img/Go.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="flagButton">
            <property name="text">
             <string>&amp;Lap</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/img/img/lap.png</normaloff>:/img/img/lap.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="resetButton">
            <property name="text">
             <string>&amp;Reset</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/img/img/reset.png</normaloff>:/img/img/reset.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
